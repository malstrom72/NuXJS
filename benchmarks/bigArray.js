var a = []; 
f=function(n) { for (var i = 0; i < n; ++i) a[i] = "" + i * 1234; };
print("1st phase");
for (var j = 0; j < 2; ++j) f(500000);
for (var i = 0; i < 500000; i += 10000) print(i + ': ' + a[i]);
var b = []; 
f=function(n) { for (var i = 0; i < n; ++i) b[i] = "" + i * 1234; };
print("2nd phase");
for (var j = 0; j < 500; ++j) f(10000);
