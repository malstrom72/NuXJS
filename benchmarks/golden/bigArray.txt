1st phase
0: 0
10000: 12340000
20000: 24680000
30000: 37020000
40000: 49360000
50000: 61700000
60000: 74040000
70000: 86380000
80000: 98720000
90000: 111060000
100000: 123400000
110000: 135740000
120000: 148080000
130000: 160420000
140000: 172760000
150000: 185100000
160000: 197440000
170000: 209780000
180000: 222120000
190000: 234460000
200000: 246800000
210000: 259140000
220000: 271480000
230000: 283820000
240000: 296160000
250000: 308500000
260000: 320840000
270000: 333180000
280000: 345520000
290000: 357860000
300000: 370200000
310000: 382540000
320000: 394880000
330000: 407220000
340000: 419560000
350000: 431900000
360000: 444240000
370000: 456580000
380000: 468920000
390000: 481260000
400000: 493600000
410000: 505940000
420000: 518280000
430000: 530620000
440000: 542960000
450000: 555300000
460000: 567640000
470000: 579980000
480000: 592320000
490000: 604660000
2nd phase